README.md

```diff
@@ -25,7 +25,14 @@ Current build status
 ====================
 
 
-<table>
+<table><tr>
+    <td>Travis</td>
+    <td>
+      <a href="https://travis-ci.com/conda-forge/cudatoolkit-feedstock">
+        <img alt="macOS" src="https://img.shields.io/travis/com/conda-forge/cudatoolkit-feedstock/master.svg?label=macOS">
+      </a>
+    </td>
+  </tr>
```

Comment:
Not sure why this appeared other than maybe from running `conda smithy rerender` on my mac

If I need to drop this and rerender using the bot LMK

Reply:
Will ask the bot to rerender once CI is done